src/js/containers/covid19/awardSpendingAgency/AwardSpendingAgencyTableContainer.jsx

```diff
@@ -0,0 +1,222 @@
+                spending_type: 'award'
+            };
+        } else {
+            params = {
+                filter: {
+                    def_codes: defCodes.map((defc) => defc.code),
+                    award_type_codes: awardTypeGroups[props.type]
+                },
+                pagination: {
+                    limit: pageSize,
+                    page: currentPage,
+                    sort: awardSpendingAgencyTableColumnFieldMapping[sort],
+                    order
+                },
+                spending_type: 'award'
+            };
+        }
+
+        let faceValueOfLoansRequest;
+        if (props.type === 'loans') {
+            const faceValueOfLoansParams = {
+                filter: {
+                    def_codes: defCodes.map((defc) => defc.code)
+                },
+                pagination: {
+                    limit: pageSize,
+                    page: currentPage,
+                    sort: awardSpendingAgencyTableColumnFieldMapping[sort],
+                    order
+                },
+                spending_type: 'award'
+            };
+            faceValueOfLoansRequest = fetchFaceValueOfLoans(faceValueOfLoansParams);
+        }
+
+
+        if (faceValueOfLoansRequest) {
+            const request = fetchAwardSpendingByAgency(params);
+
+            // merge request arrays with face_value_of_loan data
+            Promise.all([request.promise, faceValueOfLoansRequest.promise]).then((res) => {
+                const result = res[0].data.results.filter((o1) =>
+                    res[1].data.results.some((o2) =>
+                        o1.id === o2.id
+                    )
+                ).map((o) => {
+                    const faceValueOfLoan = res[1].data.results.filter((item) => o.id === item.id)[0].face_value_of_loan;
+                    return { ...o, face_value_of_loan: faceValueOfLoan };
+                });
+                parseAwardSpendingByAgency(result);
+                setTotalItems(res[0].data.page_metadata.total);
+                setLoading(false);
+                setError(false);
+            }).catch((err) => {
+                setError(true);
+                setLoading(false);
+                console.error(err);
+            });
+        } else {
+            const request = fetchAwardSpendingByAgency(params);
+            request.promise
+                .then((res) => {
+                    parseAwardSpendingByAgency(res.data.results);
+                    setTotalItems(res.data.page_metadata.total);
+                    setLoading(false);
+                    setError(false);
+                }).catch((err) => {
+                    setError(true);
+                    setLoading(false);
+                    console.error(err);
+                });
+        }
+    });
+
+    useEffect(() => {
+        // Reset to the first page
+        changeCurrentPage(1);
+        fetchSpendingByCategoryCallback();
+    }, [props.type, pageSize, sort, order]);
```

Comment:
`defCodes` is a dependency here too (doesn't matter as much without the DEFC filter in MVP, but would be helpful to add now)